### PR TITLE
OSDOCS-2995: Updated a note in ROSA policy documentation

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR updates a note in the ROSA policy documentation to align with OSD.

JIRA: https://issues.redhat.com/browse/OSDOCS-2995

Preview: https://deploy-preview-39393--osdocs.netlify.app/openshift-rosa/latest/rosa_policy/rosa-service-definition#rosa-sdpolicy-compute_rosa-service-definition

### CODE REVIEW

**Review by wgordon17 — APPROVED** [review]
/lgtm

File: modules/rosa-sdpolicy-account-management.adoc [hunk 1]
```diff
@@ -49,7 +49,9 @@ Control plane and infrastructure nodes are deployed and managed by Red Hat. Shut
 
 [NOTE]
 ====
-1 vCPU core and 1 GiB of memory are reserved on each worker node to run processes required as part of the managed service. This includes, but is not limited to, audit log aggregation, metrics collection, DNS, image registry, and SDN.
+Approximately 1 vCPU core and 1 GiB of memory are reserved on each worker node and removed from allocatable resources. This allocation of resources is necessary to run processes required by the underlying platform. These processes include system daemons such as udev, kubelet, and container runtime among others. It also accounts for kernel reservations. {OCP} core systems such as audit log aggregation, metrics collection, DNS, image registry, SDN, and others might consume additional allocatable resources to maintain the stability and maintainability of the cluster. The additional resources consumed might vary based on usage.
```
Comment (jeana-redhat) [inline modules/rosa-sdpolicy-account-management.adoc:52]: suggest s/1 vCPU core/one vCPU core

Comment (jeana-redhat) [inline modules/rosa-sdpolicy-account-management.adoc:52]: > Approximately 1 vCPU core and 1 GiB of memory are reserved on each worker node and removed from allocatable resources. This allocation of resources is necessary to run processes required by the underlying platform.

There is nothing wrong with this, but I wonder if s/allocation/reservation might make it more clear which pool is which since "allocatable" is referring to the non-reserved resources.

Comment (jeana-redhat) [inline modules/rosa-sdpolicy-account-management.adoc:52]: > It also accounts for kernel reservations.

"It" = "the reserved resources"?

**Review by jeana-redhat — COMMENTED** [review]
Just a couple small points of clarification, overall great :) One other suggestion: maybe split into two paragraphs if you can find a logical break? since note text is small it feels a bit hard to parse a longer paragraph in the output to me
